terraform-modules/aws/nlb/variables.tf

```diff
@@ -0,0 +1,110 @@
+  type        = string
+  default     = null
+}
+
+variable "enable_custom_nlb_access_logs_s3_prefix" {
+  description = "Set to true to use the value of alb_access_logs_s3_prefix for access logs prefix. If false, the alb_name will be used. This is useful if you wish to disable the S3 prefix. Only used if var.enable_alb_access_logs is true."
+  type        = bool
+  default     = false
+}
+
+variable "access_logs_s3_bucket_name" {
+  description = "The name to use for the S3 bucket where the ALB access logs will be stored. If you set this to null, a name will be generated automatically based on var.alb_name."
+  type        = string
+  default     = null
+}
+
+variable "should_create_access_logs_bucket" {
+  description = "If true, create a new S3 bucket for access logs with the name in var.access_logs_s3_bucket_name. If false, assume the S3 bucket for access logs with the name in  var.access_logs_s3_bucket_name already exists, and don't create a new one. Note that if you set this to false, it's up to you to ensure that the S3 bucket has a bucket policy that grants Elastic Load Balancing permission to write the access logs to your bucket."
+  type        = bool
+  default     = true
+}
+
+variable "num_days_after_which_archive_log_data" {
+  description = "After this number of days, log files should be transitioned from S3 to Glacier. Enter 0 to never archive log data."
+  type        = number
+}
+
+variable "num_days_after_which_delete_log_data" {
+  description = "After this number of days, log files should be deleted from S3. Enter 0 to never delete log data."
+  type        = number
+}
+
+variable "force_destroy" {
+  description = "A boolean that indicates whether the access logs bucket should be destroyed, even if there are files in it, when you run Terraform destroy. Unless you are using this bucket only for test purposes, you'll want to leave this variable set to false."
+  type        = bool
+  default     = false
+}
+
+variable "nlb_subnets" {
+    type = list(string)
+    description = "NLB Subnets"  
+}
+
+variable "enable_alb_access_logs" {
+  description = "Set to true to enable the ALB to log all requests. Ideally, this variable wouldn't be necessary, but because Terraform can't interpolate dynamic variables in counts, we must explicitly include this. Enter true or false."
+  type        = bool
+  default     = false
+}
+
+variable "enable_cross_zone_load_balancing" {
+  description = "Set enable_cross_zone_load_balancing"
+  type = bool
+  default = false
+}
+
+variable "enable_http2" {
+    description = "enable_http2"
+    type = bool
+    default = false 
+}
+
+
+variable "nlb_access_logs_s3_prefix" {
+    description = "nlb_access_logs_s3_prefix"
+    type = string
+    default = null  
+}
+
+variable "nlb_s3_bucket_name" {
+    description = "nlb_s3_bucket_name"
+    type = string
+    default = null   
```

Comment:
Need to run `terraform fmt` on this file as well.

Reply:
fixed